mne/gui/_ieeg_locate_gui.py

```diff
@@ -214,9 +221,13 @@ def _load_image_data(self, ct):
         self._ras_vox_t = np.linalg.inv(self._vox_ras_t)
 
         self._voxel_sizes = np.array(self._mri_data.shape)
-        self._img_ranges = [[0, self._voxel_sizes[1], 0, self._voxel_sizes[2]],
-                            [0, self._voxel_sizes[0], 0, self._voxel_sizes[2]],
-                            [0, self._voxel_sizes[0], 0, self._voxel_sizes[1]]]
+        self._img_ranges = [
```

Comment:
This maybe should be reverted. I get where your coming from and it does make sense but this matches `freeview`. I might be wrong though...